README.md

```diff
@@ -84,6 +84,11 @@ grunt.initConfig({
 });
 ```
 
+You can also inspect [`jsx` files used in React.js](http://facebook.github.io/react/docs/jsx-in-depth.html),
+simply by using that suffix in the `src` list.
+The command line version of `jsinspect` uses a specific flag for this,
+[namely `--jsx`](https://github.com/danielstjules/jsinspect#usage).
+
```

Comment:
I am assuming that once users get comfortable with using Grunt, they might have the curiosity to look deeper...

Unless there is anything else, let's Go!